pkg/utils/imagefilterlist_test.go

```diff
@@ -30,52 +30,67 @@ func TestSpecificImagesMustBeAccepted(t *testing.T) {
 }
 
 func TestDefaultRegistry(t *testing.T) {
-	allowedImageList := []string{"{docker.io/[AB]/some-image,some-image}", "my-user/my-image:1.2.3"}
+	allowedImageList := []string{"docker.io/[AB]/some-image", "docker.io/my-user/my-image:1.2.3"}
 	helper, err := NewImageFilterList(allowedImageList)
 	require.NoError(t, err)
 
-	assert.True(t, helper.Contains("some-image"))
-	assert.False(t, helper.Contains("B/some-image"))
-	assert.False(t, helper.Contains("some-image:123"))
-	assert.False(t, helper.Contains("B/some-image:123"))
 	assert.True(t, helper.Contains("docker.io/A/some-image"))
 	assert.True(t, helper.Contains("docker.io/B/some-image"))
+	assert.True(t, helper.Contains("docker.io/my-user/my-image:1.2.3"))
 	assert.False(t, helper.Contains("docker.io/my-user/some-image"))
-	assert.False(t, helper.Contains("docker.io/my-user/my-image:1.2.3"))
 
+	// Do not allow images without a registry
+	assert.False(t, helper.Contains("B/some-image"))
+	assert.False(t, helper.Contains("some-image:123"))
+	assert.False(t, helper.Contains("B/some-image:123"))
+	assert.False(t, helper.Contains("my-user/some-image"))
+	assert.False(t, helper.Contains("my-user/my-image:1.2.3"))
+
+	// Do not allow images from other registries
 	assert.False(t, helper.Contains("ghcr.io/my-user/some-image"))
 	assert.False(t, helper.Contains("ghcr.io/my-user/my-image:latest"))
 }
 
-func TestAcceptEverythingFromDefaultRegistry(t *testing.T) {
+func TestWildcardFormatsRegistry(t *testing.T) {
 	allowedImageList := []string{"docker.io/*", "*[AB]/some-image*"}
```

Comment:
Let's add some wildcard on the tag ;)
```suggestion
	allowedImageList := []string{"docker.io/*", "*[AB]/some-image*", "my-docker-mirror.myorg/loadimpact/k6:0.34.*"}
```